spring-cloud-contract-tools/spring-cloud-contract-maven-plugin/src/main/java/org/springframework/cloud/contract/maven/verifier/GenerateTestsMojo.java

```diff
@@ -21,6 +21,7 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+
```

Comment:
Can we revert these changes?

Reply:
Sure, but this was the effect of formatting it with the official Spring project IDE formatter.